great_expectations/data_context/store/html_site_store.py

```diff
@@ -478,3 +478,11 @@ def _unzip_assets(self, assets_full_path: str, unzip_directory: str) -> bool:
     @property
     def config(self) -> dict:
         return self._config
+
+    def self_check(self, pretty_print: bool = True) -> dict:
+        report_object = self._config
+
+        # Chetan - 20200126 - The actual pretty printing and self check mechanism is
+        # open to implement. This is simply added to adhere to `test_test_yaml_config_supported_types_have_self_check`
+
+        return report_object
```

Comment:
This was actually missing so I fixed the bug in this PR for simplicity's sake. Let me know if you'd like me to split it out

Reply:
I think you should keep it in this PR so that the new test will pass. Also good catch, that test is already paying dividends.